### ISSUE TITLE
Installation includes dev directory

### ISSUE THREAD

**Issue by awvwgk** [qcscine/chemoton#2]
Running pip install will incorrectly include the `dev` directory

```
  running install_egg_info
  Copying scine_chemoton.egg-info to build/bdist.linux-x86_64/wheel/scine_chemoton-2.1.0-py3.10.egg-info
  running install_scripts
  adding license file "LICENSE.txt" (matched pattern "LICEN[CS]E*")
  creating build/bdist.linux-x86_64/wheel/scine_chemoton-2.1.0.dist-info/WHEEL
  creating '/tmp/pip-wheel-twm84qyb/scine_chemoton-2.1.0-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
  adding 'dev/__init__.py'
  adding 'dev/conan/__init__.py'
  adding 'dev/conan/base.py'
  adding 'dev/conan/utils.py'
```

**Comment by steinmig** [comment]
I can't reproduce this, both installing from sources and from pypi does not give me the dev directory in the installed package.

Does 
```diff
     python_requires=">={}".format(".".join(str(n) for n in min_version)),
-    packages=find_packages(exclude=["docs", "tests"]),
+    packages=find_packages(exclude=["docs", "tests", "dev"]),
     entry_points={
```
in setup.py resolve your issue?

**Comment by awvwgk** [comment]
Not initializing the submodules also seems to work.

**Comment by steinmig** [comment]
Yes, not having the files present certainly prevents them from being copied^^

I thought that you meant `dev` was installed within the chemoton package, but now noticed that we indeed unknowingly also install a python package called `dev` due to our conan code being in the submodule. This should now fix your issue and will be included in the next release

```diff
      python_requires=">={}".format(".".join(str(n) for n in min_version)),
-     packages=find_packages(exclude=["docs", "tests"]),
+     packages=find_packages(include=["scine_chemoton", "scine_chemoton.*"],
+                            exclude=["scine_chemoton.tests*"]),
      entry_points={
```

**Comment by weymutht** [comment]
This is fixed with release 3.0.0.
